SofascoreAcademy/SofascoreAcademy/Views/MainViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  MainViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 08.04.2024..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+protocol parentSportSlugPicker {
+    func getPressedTab(request: sport_slug?)
+}
+
+protocol settingsButtonTap {
+    func reactToSetingsTap()
+}
+
+protocol tableCellTap {
+    func reactToCellTap(match: matchData)
```

Comment:
```suggestion
protocol tableCellTap {

    func reactToCellTap(match: matchData)
```

Razmak na pocetku, vidim da primjenjujes na klasama, mozes i tu